db/migrate/20180529225807_change_spotlight_pages_content_column_to_medium_text.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeSpotlightPagesContentColumnToMediumText < ActiveRecord::Migration[5.2]
```

Comment:
Just one stupid question; will specifying `[5.2]` here mean that Rails 5.1 won't be able to run the migration?

Reply:
Answer: yes; Rails refuses to run a migration that's tagged with a greater version. I think we need to specify `5.1` instead.

Reply:
Yeah, I just did whatever rails generating for me w/o thinking too much about it. 😳